lib/Runtime/Library/BoundFunction.cpp

```diff
@@ -215,7 +215,9 @@ namespace Js
                 newValues[index++] = args[i];
             }
 
-            if (args.HasExtraArg())
+            // Check for CallFlags_ExtraArg to be consistent with GetLargeArgCountWithExtraArgs.
+            // Do not use 'HasExtraArg' since that also considers 'CallFlags_NewTarget', but GetLargeArgCountWithExtraArgs does not.
```

Comment:
This sounds like a bug in `GetLargeArgCountWithExtraArgs`... There are a few different times that we have an extra arg, and having a newtarget is one of those cases (but my understanding is that they should be mutually exclusive cases, so we only ever need at most one extra arg). Would it make more sense to update `GetLargeArgCountWithExtraArgs` to match `HasExtraArg`?

Reply:
I was thinking the same way, but was not sure.